src/Jackett.Common/Definitions/insanetracker.yml

```diff
@@ -95,51 +97,48 @@
         selector: a.cover
         attribute: href
       files:
-        selector: td:nth-child(6)
+        selector: div.files > a.button.small
         filters:
           - name: replace
             args: [".", ""]
       size:
-        selector: td:nth-child(8)
+        selector: td.center.size
         filters:
           - name: replace
             args: [".", ""]
           - name: replace
             args: [",", "."]
-      grabs:
-        selector: td:nth-child(9) > div:first-child
+      seeders:
+        selector: td.center.data > a:first-child
         filters:
           - name: replace
             args: [".", ""]
-          - name: regexp
-            args: "^([\\d]+)"
-      seeders:
-        selector: td:nth-child(9) > div:first-child
+      leechers:
+        selector: td.center.data > a:nth-child(2)
         filters:
           - name: replace
             args: [".", ""]
-          - name: regexp
-            args: "\\|\\s*([\\d]+)\\s*\\|"
-      leechers:
-        selector: td:nth-child(9) > div:first-child
+      grabs:
+        selector: td.center.data
         filters:
           - name: replace
             args: [".", ""]
-          - name: regexp
-            args: "([\\d]+)$"
+        remove: div.uploader, div.moderation, a.leftborder
+
       date:
-        selector: td.date
+        selector: td.center.date
         filters:
+          - name: re_replace
+            args: ["(\\-)(\\d{2})(\\d{2})(\\:)", "-$2 $3:"]
           - name: dateparse
-            args: "2006.01.02 15:04:05"
+            args: "2006-01-02 08:01"
```

Comment:
dateparse uses the GoLang layout. 
see https://github.com/Jackett/Jackett/wiki/Definition-format#dateparse for details.
I expect the hours:minutes should probably be 15:04